pkg/api/types.go

```diff
@@ -1353,14 +1353,14 @@ type EndpointsList struct {
 
 // NodeSpec describes the attributes that a node is created with.
 type NodeSpec struct {
-	// PodCIDR represents the pod IP range assigned to the node
+	// PodCIDR represents the pod IP range assigned to the node.
```

Comment:
nit: peel these off into another PR?  They're orthogonal to the point of this PR and easy to review.